src/System.Management.Automation/DscSupport/CimDSCParser.cs

```diff
@@ -41,9 +41,9 @@ public static object ConvertCimInstanceToObject(Type targetType, CimInstance ins
 
             using (System.Management.Automation.PowerShell powerShell = System.Management.Automation.PowerShell.Create(RunspaceMode.CurrentRunspace))
```

Comment:
In general, that is a good point; however:
1) this code was (working fine) like this since forever; to avoid regressions it is better not to touch this.
2) these (essentially utility) methods were made public just for ease of access from DSC module, they were never meant to be used by public. Not the best design, I agree, but it is what it is.
3) all this will be gone anyway soon when this is moved to a subsystem in DSC module.